.eslintrc.json

```diff
@@ -0,0 +1,75 @@
+{
+  "env": {
+      "browser": true,
+      "es6": true
+  },
+  "extends": "airbnb",
+  "parser": "babel-eslint",
+  "globals": {
+    "window": false,
+    "navigator": false,
+    "document": false,
+    "location": false,
+    "Atomics": "readonly",
+    "SharedArrayBuffer": "readonly"
+  },
+  "parserOptions": {
+      "ecmaFeatures": {
+          "jsx": true
+      },
+      "ecmaVersion": 2018,
+      "sourceType": "module"
+  },
+  "plugins": [
+      "react"
+  ],
+  "rules": {
+    "consistent-return": "off",
+    "no-plusplus": "off",
+    "no-mixed-operators": "off",
+    "no-bitwise": "off",
+    "no-underscore-dangle": "off",
+    "no-multiple-empty-lines": "off",
+    "no-unused-expressions": "off",
+    "no-new": "warn",
+    "no-eval": "warn",
+    "no-lone-blocks": "off",
+    "no-return-assign": "off",
+    "class-methods-use-this": "off",
+    "import/prefer-default-export": "off",
+    "prefer-rest-params": "warn",
+    "react/no-string-refs": "off",
+    "react/require-default-props": "off",
+    "react/no-unused-prop-types": "off",
+    "react/no-deprecated": "off",
+    "react/prefer-stateless-function": "warn",
+    "comma-dangle": ["error", "always-multiline"],
+    "valid-jsdoc": ["error", {
+      "requireReturn": false,
+      "requireParamDescription": false,
+      "requireReturnDescription": false
+    }],
+    "no-restricted-properties": ["off", {
+        "object": "Math",
+        "property": "pow"
+    }],
+    "no-continue": "off",
+    "max-len": "off",
+    "no-param-reassign": "off",
+    "new-cap": "off",
+    "no-use-before-define": "off",
+    "object-curly-newline" : "off"
+  },
```

Comment:
While it's not one of our normal ones (though several of us are in favor of it) and since this project should already be following it, please add `curly` at warn or error levels.

Reply:
Done

Reply:
Not seeing `curly` in eslintrc, just `object-curly-newline` which is a different rule.